scripts/magpietts/infer_and_evaluate_streaming.py

```diff
@@ -0,0 +1,566 @@
+
+    if args.datasets is None:
+        args.datasets = EVALUATION_DATASETS
+
+    # FCD computation is enabled by default, disabled only when --disable_fcd is specified
+    compute_fcd = not args.disable_fcd
+
+    estimate_alignment_from_layers = None
+    if args.estimate_alignment_from_layers is not None:
+        estimate_alignment_from_layers = [int(l.strip()) for l in args.estimate_alignment_from_layers.split(",")]
+    apply_prior_to_layers = None
+    if args.apply_prior_to_layers is not None:
+        apply_prior_to_layers = [int(l.strip()) for l in args.apply_prior_to_layers.split(",")]
+
+    run_inference_w_args = partial(
+        run_inference_streaming,
+        datasets=args.datasets.split(","),
+        out_dir=args.out_dir,
+        temperature=args.temperature,
+        topk=args.topk,
+        codecmodel_path=args.codecmodel_path,
+        use_cfg=args.use_cfg,
+        cfg_scale=args.cfg_scale,
+        batch_size=args.batch_size,
+        sv_model=args.sv_model,
+        asr_model_name=args.asr_model_name,
+        num_repeats=args.num_repeats,
+        apply_attention_prior=args.apply_attention_prior,
+        attention_prior_epsilon=args.attention_prior_epsilon,
+        attention_prior_lookahead_window=args.attention_prior_lookahead_window,
+        estimate_alignment_from_layers=estimate_alignment_from_layers,
+        apply_prior_to_layers=apply_prior_to_layers,
+        start_prior_after_n_audio_steps=args.start_prior_after_n_audio_steps,
+        confidence_level=args.confidence_level,
+        use_local_transformer=args.use_local_transformer,
+        maskgit_n_steps=args.maskgit_n_steps,
+        legacy_codebooks=args.legacy_codebooks,
+        legacy_text_conditioning=args.legacy_text_conditioning,
+        clean_up_disk=args.clean_up_disk,
+        hparams_file_from_wandb=args.hparams_file_from_wandb,
+        log_exp_name=args.log_exp_name,
+        compute_fcd=compute_fcd,
+        violin_plot_metrics=args.violin_plot_metrics,
+    )
+
+    # Mode 1: Run inference from provided hparams and checkpoint files
+    if (
+        (args.hparams_files is not None)
+        and (args.checkpoint_files is not None)
+        and (args.hparams_files != "null")
+        and (args.checkpoint_files != "null")
+    ):
+        hparam_files = args.hparams_files.split(",")
+        checkpoint_files = args.checkpoint_files.split(",")
+        print("Running inference for hparams files: ", hparam_files)
+        print("Running inference for checkpoint files: ", checkpoint_files)
+        assert len(hparam_files) == len(
+            checkpoint_files
+        ), "Number of hparams files and checkpoint files should be the same."
+        for hparams_file, checkpoint_file in zip(hparam_files, checkpoint_files):
+            cer, ssim = run_inference_w_args(
+                hparams_file=hparams_file,
+                checkpoint_file=checkpoint_file,
+                nemo_file=None,
+            )
+        return
```

Comment:
we don't have to `return` here right? it looks like [infer_and_evaluate.py](https://github.com/NVIDIA-NeMo/NeMo/blob/magpitts_2503_small/scripts/magpietts/infer_and_evaluate.py#L338) also return here. @blisc any comments?

Reply:
Good catch. This looks like a bug in [infer_and_evaluate.py](https://github.com/NVIDIA-NeMo/NeMo/blob/magpitts_2503_small/scripts/magpietts/infer_and_evaluate.py#L338). @subhankar-ghosh Can you remove the return statement from both scripts?

Reply:
I removed `return` for both.